packages/salesforcedx-vscode-apex/src/telemetry/telemetry.ts

```diff
@@ -79,6 +87,29 @@ export class TelemetryService {
     }
   }
 
+  public sendErrorEvent(
+    error: { message: string; stack?: string },
+    additionalData?: any,
```

Comment:
I think commandName can be sent as part of additionalData.

Also, is it work deleting the `sendError` error method and update the code to use the new `sendErrorEvent` ?

Reply:
You mean `sendError` in the core module? It has quite a few references, but not a ridiculous amount. I will create separate work for it. I'll modify additionalData though 👍